x/app-router/types/parser.go

```diff
@@ -0,0 +1,50 @@
+package types
+
+import (
+	"strings"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+type ParsedReceiver struct {
+	ShouldLiquidStake bool
+	StrideAccAddress  sdk.AccAddress
+}
+
+// {stride_address}|{module_id}/{function_id}
+// for now, only allow liquid staking transactions
+// TODO: consider - addresses that previously *could not* liquid stake, now *can*
```

Comment:
I think this could be answered by Aidan - he added the comment - is it for module account probably?
(User can send to module account via IBC and let it to stake)

Reply:
@asalzmann